vulcan/lib/server/workflows/scripts/parse_record_selections.py

```diff
@@ -1,34 +1,42 @@
-from archimedes.functions.dataflow import output_json, input_json
+from archimedes.functions.dataflow import output_json, input_json, parseModelAttr, buildTargetPath
 from archimedes.functions.magma import connect, question
 from archimedes.functions.list import unique, flatten
+from archimedes.functions.environment import project_name
 
+pdat = input_json("project_data")[project_name]
+selection_options = list(pdat['selection_options'].values())
 
-seq_model_name = 'sc_seq'
-seq_pool_model_name = 'sc_seq_pool'
 magma = connect()
 
-experiments = input_json('experiments')
-tissues = input_json('tissues')
-fractions = input_json('fractions')
+select1 = input_json('select1')
+select2 = input_json('select2')
+select3 = input_json('select3')
```

Comment:
Does this mean projects will only be limited to 3 record selection methods? Is that normal -- not sure if you think some might have more?